php_phongo_structs.h

```diff
@@ -43,6 +43,12 @@ typedef struct {
 	PHONGO_ZEND_OBJECT_POST
 } php_phongo_bulkwrite_t;
 
+typedef struct {
+	PHONGO_ZEND_OBJECT_PRE
+	mongoc_client_encryption_t* client_encryption;
```

Comment:
Sorry to bring this up, but I think we may need to consider how forking might affect FLE. Consider that this has a reference to two clients (its own and the key vault), and internal operations with the key vault client don't use our API that might conditionally call `mongoc_client_reset` when needed. I'm not sure how often the key vault client is used over the lifetime of this object, but perhaps we'll need to consider PID checks like we do for cursors and sessions when certain methods are called or this is destroyed.

There's probably a more general concern about auto encryption if the key vault is used over the lifetime of an application, too. I'm not going to insist we sort this out now (or even in 1.7.0), but would you mind opening a ticket and referring back to our FLE epic and [PHPC-1274](https://jira.mongodb.org/browse/PHPC-1274)?

Reply:
Noting that [PHPC-1529](https://jira.mongodb.org/browse/PHPC-1529) has been opened to track forking and FLE.